### PR TITLE
feat: named mapping params

### PR DESCRIPTION
**Pull request by 0xClandestine**

### CODE REVIEW

**Comment by 0xClandestine** [comment]
Storage CI failing because `beaconChainETHSharesToDecrementOnWithdrawal` has been renamed to `__deprecated_beaconChainETHSharesToDecrementOnWithdrawal` since it was deprecated but not renamed.

File: src/contracts/core/DelegationManagerStorage.sol [hunk 1]
```diff
@@ -68,73 +68,51 @@ abstract contract DelegationManagerStorage is IDelegationManager {
...
-    /// @notice Mapping: hash of withdrawal inputs, aka 'withdrawalRoot' => whether the withdrawal is pending
-    mapping(bytes32 => bool) public pendingWithdrawals;
+    /// @notice Returns whether a given `withdrawalRoot` has a pending withdrawal.
+    mapping(bytes32 withdrawalRoot => bool pending) public pendingWithdrawals;
 
-    /// @notice Mapping: staker => cumulative number of queued withdrawals they have ever initiated.
+    /// @notice Returns the total number of withdrawals that have been queued for a given `staker`.
     /// @dev This only increments (doesn't decrement), and is used to help ensure that otherwise identical withdrawals have unique hashes.
-    mapping(address => uint256) public cumulativeWithdrawalsQueued;
+    mapping(address staker => uint256 total) public cumulativeWithdrawalsQueued;
```
Comment (ypatil12) [inline src/contracts/core/DelegationManagerStorage.sol:104]: Nit: maybe `totalQueued`?

File: src/contracts/core/RewardsCoordinatorStorage.sol [hunk 2]
```diff
@@ -78,27 +79,27 @@ abstract contract RewardsCoordinatorStorage is IRewardsCoordinator {
...
-    /// @notice Mapping: earner => the address of the entity who can call `processClaim` on behalf of the earner
-    mapping(address => address) public claimerFor;
+    /// @notice Returns the `claimer` for a given `earner`.
+    /// @dev The claimer is able to call `processClaim` on behalf of the `earner`.
+    mapping(address earner => address claimer) public claimerFor;
 
-    /// @notice Mapping: earner => token => total amount claimed
-    mapping(address => mapping(IERC20 => uint256)) public cumulativeClaimed;
+    /// @notice Returns the total claimed amount for an `earner` for a given `token`.
+    mapping(address earner => mapping(IERC20 token => uint256 claimed)) public cumulativeClaimed;
```
Comment (ypatil12) [inline src/contracts/core/RewardsCoordinatorStorage.sol:87]: Nit: `totalClaimed`

File: src/contracts/pods/EigenPodManagerStorage.sol [hunk 3]
```diff
@@ -73,7 +73,7 @@ abstract contract EigenPodManagerStorage is IEigenPodManager {
      * Likewise, when a withdrawal is completed, this "deficit" is decreased and the withdrawal amount is decreased; We can think of this
      * as the withdrawal "paying off the deficit".
      */
-    mapping(address => int256) public podOwnerDepositShares;
+    mapping(address podOwner => int256) public podOwnerDepositShares;
```
Comment (ypatil12) [inline src/contracts/pods/EigenPodManagerStorage.sol:76]: shares?

File: src/contracts/core/AVSDirectoryStorage.sol [hunk 4]
```diff
@@ -41,40 +41,37 @@ abstract contract AVSDirectoryStorage is IAVSDirectory {
...
-    /// @dev This storage will be deprecated once M2 based deregistration is deprecated.
-    mapping(address => mapping(address => OperatorAVSRegistrationStatus)) public avsOperatorStatus;
+    /// @notice Returns the registration status of each `operator` for a given `avs`.
+    /// @dev This storage will be deprecated once M2-based deregistration is removed.
+    mapping(address avs => mapping(address operator => OperatorAVSRegistrationStatus)) public avsOperatorStatus;
 
-    /// @notice Mapping: operator => salt => Whether the salt has been used or not.
-    mapping(address => mapping(bytes32 => bool)) public operatorSaltIsSpent;
+    /// @notice Returns whether a `salt` has been used by a given `operator`.
+    mapping(address operator => mapping(bytes32 salt => bool)) public operatorSaltIsSpent;
```
Comment (ypatil12) [inline src/contracts/core/AVSDirectoryStorage.sol:52]: `bool isSpent` ?

File: src/contracts/core/AVSDirectoryStorage.sol [hunk 5]
```diff
@@ -41,40 +41,37 @@ abstract contract AVSDirectoryStorage is IAVSDirectory {
...
+    /// @notice Returns the list of `operators` that are members of a given operator set.
+    /// @dev Each key is formatted as `bytes32(abi.encodePacked(avs, uint96(operatorSetId)))`.
+    mapping(bytes32 operatorSetKey => EnumerableSet.AddressSet) internal _operatorSetMembers;
 
-    /// @notice Mapping: operatorSet => List of strategies that the operatorSet contains
-    /// @dev Each key is formatted as such: bytes32(abi.encodePacked(avs, uint96(operatorSetId)))
-    mapping(bytes32 => EnumerableSet.AddressSet) internal _operatorSetStrategies;
+    /// @notice Returns the list of `strategies` associated with a given operator set.
+    /// @dev Each key is formatted as `bytes32(abi.encodePacked(avs, uint96(operatorSetId)))`.
+    mapping(bytes32 operatorSetKey => EnumerableSet.AddressSet) internal _operatorSetStrategies;
```
Comment (ypatil12) [inline src/contracts/core/AVSDirectoryStorage.sol:70]: Can we name these custom types. So `strategies` here?

**Review by ypatil12 — COMMENTED** [review]
Minor nits

**Review by ypatil12 — APPROVED** [review]

**Pull request merged**